tracr/rasp/rasp.py

```diff
@@ -157,10 +157,9 @@ def annotated(self: RASPExprT, **annotations) -> RASPExprT:
 
 def annotate(expr: RASPExprT, **annotations) -> RASPExprT:
   """Creates a new expr with added annotations."""
```

Comment:
Should change this comment to reflect the new behavior (does not create a new expr anymore)